src/colmap/scene/database.cc

```diff
@@ -716,12 +716,18 @@ PosePrior Database::ReadPosePrior(const image_t image_id) const {
   PosePrior prior;
   const int rc = SQLITE3_CALL(sqlite3_step(sql_stmt_read_pose_prior_));
   if (rc == SQLITE_ROW) {
-    prior.position =
-        ReadStaticMatrixBlob<Eigen::Vector3d>(sql_stmt_read_pose_prior_, rc, 1);
     prior.coordinate_system = static_cast<PosePrior::CoordinateSystem>(
-        sqlite3_column_int64(sql_stmt_read_pose_prior_, 2));
-    prior.position_covariance =
-        ReadStaticMatrixBlob<Eigen::Matrix3d>(sql_stmt_read_pose_prior_, rc, 3);
+        sqlite3_column_int64(sql_stmt_read_pose_prior_, 1));
+
+    prior.SetPosition(ReadStaticMatrixBlob<Eigen::Vector3d>(
```

Comment:
I am wondering whether we want the PosePrior to store the world_from_cam pose, such that reading/writing from the database is a perfect roundtrip?